### PR TITLE
fix(clay css) #1424 Add Clay Time (Time Picker)

### PR DESCRIPTION
**Pull request by pat270**
https://issues.liferay.com/browse/LEXI-481

### CODE REVIEW

**Comment by matuzalemsteles** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: packages/clay-css/src/content/form_elements_date_picker.html [hunk 1]
```diff
@@ -162,17 +162,41 @@ <h3>Date Picker</h3>
...
+											</div>
 										</div>
 										<div class="input-group-item">
-											<input class="form-control" name="timer" type="time" value="00:00">
+											<div class="form-control">
+												<div class="clay-time-edit">
+													<input class="clay-time-hours form-control-inset" id="timePicker12hTimeZoneIcon" maxlength="2" name="uniqueNameHours" type="text" value="12">
+													<span class="clay-time-divider">:</span>
+													<input class="clay-time-minutes form-control-inset" maxlength="2" name="uniqueNameMinutes" type="text" value="00">
+													<input class="clay-time-ampm form-control-inset" maxlength="2" name="uniqueNameAmpm" type="text" value="PM">
```
Comment (matuzalemsteles) [inline packages/clay-css/src/content/form_elements_date_picker.html]: I'm wondering if it should really be an input or select, I'm not sure what the default is. But it seems strange that this is an input. Any idea?

Comment (pat270) [reply packages/clay-css/src/content/form_elements_date_picker.html]: @matuzalemsteles We could use select but the browser pops up the select menu which wasn't in the Lexicon Designs. The browser default is a text input that changes the value with the up and down arrow keys.

It doesn't matter to me what type of element we use as long as the Lexicon Team doesn't care.

Comment (matuzalemsteles) [reply packages/clay-css/src/content/form_elements_date_picker.html]: hmm ok, hey @victorvalle what do you think about it?

Comment (marcoscv-work) [reply packages/clay-css/src/content/form_elements_date_picker.html]: I think it's fine and we can go ahead and merge it! :-)

Comment (matuzalemsteles) [reply packages/clay-css/src/content/form_elements_date_picker.html]: 🙂yeah, I think we can review this later so we do not block this PR.

**Pull request merged**